### DISCUSSION TITLE
Works on current IOS 17?

### DISCUSSION THREAD

**Discussion by shiprec** [Chapoly1305/FindMy#10] [Q&A]
Hello @Chapoly1305 I wanted to ask if you are able to get location reporting from IOS 17 devices?  I have heard this has been a problem from a lot of people.

**Comment by Chapoly1305** [comment]
If you mean you have a tag (created by following the OpenHayStack for example), and wish to know the location of this tag, then yes. You may follow the video to setup a personal server to get the location of the tag. 

If you mean to track a IOS17 device, then this can't accomplish by this project. 

If you mean to track a airtag on IOS17 without using the Apple FindMy, this project can't do as well. 

If none of these, I would need your kindly a more detailed description.

**Comment by shiprec** [comment]
it seems like tags created using openhaystack get location reports sent by devices that are <IOS17.  But if the tag is near a device that is IOS17 the location reports do not get sent.  Have you seen this same issue?

**Reply by Chapoly1305** [reply]
Very interesting, I have not hear about this. May I know which device you have tested with, and is this being validated (by others)?
